services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/core/storage.py

```diff
@@ -15,30 +16,53 @@
 _LIVENESS_TIMEOUT: Final[timedelta] = timedelta(seconds=5)
 
 
-async def _is_storage_responsive(url: str) -> bool:
+def _get_auth(storage_auth_settings: StorageAuthSettings) -> tuple[str, str] | None:
```

Comment:
Maybe use a `NamedTuple`?

Reply:
Apparently it works. So let's use it.

Reply:
I will rename it locally.

I have stated above why not adding it as a member of StorageAuthSettings